### DISCUSSION TITLE
k8s-fluxcd-helm-health

### DISCUSSION THREAD

**Discussion by stewartshea** [runwhen-contrib/runwhen-local#203] [Troubleshooting Commands]
🌟Welcome to the community discussion for the troubleshooting commands defined in: [k8s-fluxcd-helm-health](https://github.com/runwhen-contrib/rw-cli-codecollection/tree/main/codebundles/k8s-fluxcd-helm-health) 

🌱This discussion is intended to help grow the community
🛠️Together, let's enhance the usefulness of these troubleshooting commands
🙌 Your participation is crucial! Share your valuable comments and questions here, and don't forget to give a thumbs up if you find the commands helpful and utilize them in your work.

**Comment by stewartshea** [comment]
In a discussion today, one idea for a task came up to figure out exactly which object is being reconciled when FluxCD says the the release is in a reconciling state. We have something like this for Kustomizations, but not HelmReleases. 

@emily-zall did I capture that correctly?

**Reply by emily-zall** [reply]
yes! thank you!
